plugins/rocks.xml.webhelp/css/layout-common.css

```diff
@@ -110,6 +110,6 @@ main {
 .footer-text {
     padding: 1rem;
     text-decoration: none;
-    color: #1a1e21;
+    color: #FFFFFF;
```

Comment:
Text color should be #b2b2b2 as far as I remember

Reply:
fixed